lib/cli/emulate.js

```diff
@@ -20,11 +20,35 @@
  */
 var self,
     open = require('open'),
-    emulate = require('./../server/emulate');
+    emulate = require('./../server/emulate'),
+    help = require('./help'),
+    colors = require('colors');
+
+colors.mode = "console";
 
 self = {
     call: function (opts) {
-        self.start(opts);
+        if (self.validateOptions(opts)) {
+            self.start(opts);
+        } else {
+            console.error(('Options not understood: ' + process.argv.splice(2).join(' ')).red.bold);
+            help.call('emulate');
+        }
+    },
+    validateOptions: function (opts) {
```

Comment:
Yeah, that's fine. As long as you have a goal to use this externally, I'm fine merging it as is.